sdk/spring/spring-cloud-azure-integration-tests/src/test/java/com/azure/spring/cloud/integration/tests/servicebus/ServiceBusIT.java

```diff
@@ -26,6 +26,7 @@
 @SpringBootTest(
     webEnvironment = SpringBootTest.WebEnvironment.NONE,
     properties = {
+        //exclude these two configurations avoiding start binder bean won't use which activated by errorhandler
```

Comment:
exclude these two configurations to avoid activating the Spring Cloud Stream Binder, which will be activated unintentionally by error handler (which is a Consumer<> bean)